go/api/sorted_set_commands.go

```diff
@@ -263,6 +263,66 @@ type SortedSetCommands interface {
 	// [blocking commands]: https://github.com/valkey-io/valkey-glide/wiki/General-Concepts#blocking-commands
 	BZPopMin(keys []string, timeoutSecs float64) (Result[KeyWithMemberAndScore], error)
 
+	// Returns the specified range of elements in the sorted set stored at `key`.
```

Comment:
Move the doc comments to client_base.go in order for them to be properly displayed by `pkgsite` and `go doc`